### PR TITLE
Update header 

### PR DESCRIPTION
**Pull request by vagita24**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,8 +28,9 @@
       <header class="text-center">
         <h1>Welcome to Public Lab!</h1>
         <p>
-          Powered by Commmunity Toolbox
-          <a href="https://github.com/jywarren/community-toolbox"><i class="fa fa-github"></i></a> Open Source
+          Powered by <a href="https://github.com/jywarren/community-toolbox"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/jywarren/community-toolbox">Commmunity Toolbox</a> 
+          | Open Source
```
Comment (gauravano) [inline index.html]: @jywarren do we need these changes?

File: index.html [hunk 2]
```diff
@@ -43,7 +44,7 @@ <h4 style="text-align:center;font-weight:bold;">We're so happy to meet you! This
 
       <p>Our community aspires to be a respectful place. Please read and abide by our <a href="https://publiclab.org/conduct">Code of Conduct</a>.</p>
 
-      <p>If you're new, and looking to get involved, you're in the right place. Check out <a href="https://github.com/publiclab/plots2/">our README</a> to get started.</p>
+      <p>If you're new, and looking to get involved, you're in the right place. Check out <a href="https://github.com/publiclab/plots2/blob/master/README.md">our README</a> to get started.</p>
```
Comment (gauravano) [inline index.html]: This one is fine!

**Comment by gauravano** [comment]
Hi @vagita24, can you link the issue you are trying to solve? Thanks!

File: index.html (hunk 1 above)
Comment (gauravano) [reply index.html]: This part is not present in main branch. Do we need this "powered by" part? @jywarren

**Comment by jywarren** [comment]
I think not -- thank you!

**Comment by gauravano** [comment]
Closing this PR as no changes are required in main after resolving the conflict. Thanks!

**Pull request closed without merging**